hack/vsa_validation_demo/README.md

```diff
@@ -0,0 +1,80 @@
+# VSA Validation Demo
```

Comment:
What's the purpose of the `vsa_validation_demo`? The service only generates one, so maybe this directory can be removed?